src/shared/locales/en/public-witnessing.json

```diff
@@ -0,0 +1,29 @@
+{
+  "tr_PWLocations": "Public witnessing locations",
+  "tr_editLocations": "Edit locations",
+  "tr_PW": "Public witnessing",
+  "tr_locationCart": "Cart on {{ locatinName }}",
+  "tr_maxPublisherShift": "Max. publishers per shift: {{ maxPublisherCount }}",
+  "tr_partnerNeeded": "Partner needed",
+  "tr_partnerNeededDesc": "Other publishers will see this and be able to arrange with you",
+  "tr_locationName": "Location name",
+  "tr_GeneralTimeRules": "General time rules",
+  "tr_maxPublisherLabel": "Max. publishers per shift ",
+  "tr_whereGetCart": "Where to get a cart",

```

Comment:
@rhahao what is your opinion? How to call a place where the stand is stored _(e.g. at a brother or at a Kingdom Hall_)? What short and generic phrasing we can use here to include everything (cart, literature table, something else) and be simple? For example, pick-up point, storage point, storage location? Any better options?

Reply:
@ux-git, @mrtavor: Let’s consider refining the naming later. In that way, we can merge this PR at this time.

Reply:
@rhahao sure 👍